### ISSUE TITLE
java.lang.NullPointerException: Unknown error produced when bundling for iOS

### ISSUE THREAD

**Issue by vgreen7** [defold/editor2-issues#2938]
### Expected behaviour
Bundle Defold project as iOS application
### Actual behaviour
reported error canceling build stating:
java.lang.NullPointerException: Unknown

![Screen Shot 2020-04-01 at 9 44 06 AM](https://user-images.githubusercontent.com/55544989/78163556-69f7f780-73fd-11ea-941f-19403df51e88.png)

### Steps to reproduce
No steps could be found to reproduce.
<hr/>

<table>
<tr><td>Build time</td><td>2020-03-30T15:27:54.357768</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>b673cc217e4b968a58c922f4b724657b4f7255c0</td></tr>
<tr><td>Defold engine sha</td><td>96f7a5e4f617d5f6f4645f30a3e6ff656689435d</td></tr>
<tr><td>Defold version</td><td>1.2.167</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"57c09dd981384b59a30c75a1afba7970"'>57c09dd981384b59a30c75a1afba7970</a></td></tr>
<tr><td>GPU</td><td>NVIDIA GeForce 9400M OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 NVIDIA-10.4.14 310.90.30.05b27</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.13.6</td></tr>
<table>

**Comment by britzl** [comment]
And this only happens when you bundle to iOS? The same project can be bundled to macOS for instance? Can you bundle an empty project to iOS?

**Comment by vgreen7** [comment]
No, I was not able to bundle to any other platform. An empty project bundled fine. 

The issue turned out to be that I called a lua script as ".onscreen" when I should have had it as "blank.onscreen." By putting onscreen.lua into a folder and calling it from there it removed the issue. 
[Forum Post](https://forum.defold.com/t/java-lang-nullpointerexception-unknown-solved/64778/12)

**Comment by britzl** [comment]
It should also work to remove the initial dot and keep the file in the root of the project. It is cleaner though to have the files in folders.
